### DISCUSSION TITLE
[QUESTION] Cómo puedo hacer un iscomanded de una componente opcion multiple tipo usuario?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#238] [2. Questions and Answers] [answered]
Hola

Quisiera saber cómo se puede realizar la siguiente accion

En usuarios, creo un usuario "**Otro**", para que la componente opción múltiple / tipo usuario, seleccione ese usuario "**Otro**" y me aparezca una componente de tipo texto para escribir esa persona.

Se que si coloco cambio el tipo a colección, puedo colocar un "Otro", y me aparezca el iscomanded, pero no lo quiero hacer, debido a que si en un futuro existe una actualización en la dotación, tendría que hacer una mantención a la colección para que no aparezca los usuarios inactivos y/o que aparezca los nuevos usuarios.

Gracias =>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG El code que retorna una question de tipo user para el isCommanded es: `${userDisplayName} <${u.email}>`

**Reply by NicolasPLG** [reply]
hola @guillevalin eso sería como colocar el correo del usuario?

**Reply by guillevalin** [reply]
En teoria si hicieras un regex por el display name o el email deberia funcionar.

**Reply by NicolasPLG** [reply]
ok gracias @guillevalin probaré con usar el email como prueba. dejaré el resultado en otra discussion con referencia a está.

Gracias

**Reply by guillevalin** [reply]
Confirmo que funciona!

![image](https://user-images.githubusercontent.com/34345309/123278126-b1701500-d4d4-11eb-8f03-b6143d3bd075.png)

![image](https://user-images.githubusercontent.com/34345309/123278186-bc2aaa00-d4d4-11eb-9f3b-33eda5a6abfd.png)

La configuración:
![image](https://user-images.githubusercontent.com/34345309/123278254-cba9f300-d4d4-11eb-8d30-d3275c992685.png)
